anchoring/index.js

```diff
@@ -207,6 +276,12 @@ const getAllVersions = (keySSI, authToken, callback) => {
     versions(keySSI, authToken, callback);
 }
 
+// This should be called "getLatestVersion" but we already have
```

Comment:
why do we need this method definition if is just an alias for the latestVersion method? maybe we should just rename it in the module.exports object getLastVersion: latestVersion.

Reply:
The only reason is consistency with the other public functions: `getAllVersions()` calls `versions()`. I'll remove the alias